translations/bnpchs-final-2019-makhluk-kaki-banyak.html

```diff
@@ -0,0 +1,107 @@
+<style>
+  table, th, td{
+  border: 1px solid black;
+  border-collapse: collapse;
+}
+</style>
+
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Feli is a resident of the Beenus kingdom that has $$$N$$$ legs. One day, the Beenus kingdom will hold a lively party. All of the resident of Beenus is invited, including Feli. To respect the party's owner, everyone should wear a polite and descent clothe.
+</p>
+
+<p style="text-align: justify">
+  Feli wants to wear a polite and descent clothe that consists of $$$N$$$ pants, $$$N$$$ socks and $$$N$$$ shoes with the following rules:
+</p>
+
+<ul>
+  <li>Shoes can be worn on the <em>i</em>-th foot if and only if a pant and a sock have been worn on the <em>i</em>-th foot</li>
+  <li>Feli is able to change the order of wearing clothes according to her heart but she still has to follow the rules</li>
+</ul>
+
+<p style="text-align: justify;">
+  When the party started, Feli suddenly has a question "How many ways are there to wear pants, socks and shoes if she still has to follow the rules above". Help Feli to answer that question. Because the answer can be very large, Feli wants the answer modulo $$$M$$$.
```

Comment:
```suggestion
  When the party started, Feli suddenly has a question "How many ways are there to wear pants, socks and shoes if she follows the rules above". Help Feli to answer that question. Because the answer can be very large, Feli wants the answer modulo $$$M$$$.
```